internal/bminventory/inventory.go

```diff
@@ -2048,7 +2050,7 @@ func (b *bareMetalInventory) GetClusterSupportedPlatformsInternal(
 		return nil, fmt.Errorf("error getting cluster, error: %w", err)
 	}
 	// no hosts or SNO
-	if len(cluster.Hosts) == 0 || swag.StringValue(cluster.HighAvailabilityMode) != models.ClusterHighAvailabilityModeFull {
+	if common.CountActiveHosts(cluster) <= 1 {
```

Comment:
I don't understand the logic here (now and before) - we're returning baremetal platform for SNO?

Reply:
Baremetal is the only platform supported for SNO, isn't it?

Reply:
Actually, none platform is the only platform supported for SNO

Reply:
None is the only supported platform for SNO

Reply:
Ouch, now I understand your first message @avishayt. Then I will check the number of enabled hosts in the cluster and if there are <= 1 then I will return 'none' platform. sounds good?

Reply:
Yes that makes sense